src/skins/vector/views/organisms/LeftPanel.js

```diff
@@ -40,7 +40,7 @@ module.exports = React.createClass({
             classes += " collapsed";
         }
         else {
-            collapseButton = <img className="mx_LeftPanel_hideButton" onClick={ this.onHideClick } src="img/hide.png" width="12" height="20" alt="<"/>   
+            //collapseButton = <img className="mx_LeftPanel_hideButton" onClick={ this.onHideClick } src="img/hide.png" width="12" height="20" alt="<"/>   
         }
```

Comment:
This `else` block is completely dead code. Can we remove it? If you intend to keep it, please explain why in a comment.